pkg/ccl/changefeedccl/alter_changefeed_test.go

```diff
@@ -1346,3 +1261,52 @@ func TestAlterChangefeedUpdateFilter(t *testing.T) {
 
 	cdcTest(t, testFn, feedTestEnterpriseSinks)
 }
+
+func TestAlterChangefeedInitialScan(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	testFn := func(initialScanOption string) cdcTestFn {
+		return func(t *testing.T, db *gosql.DB, f cdctest.TestFeedFactory) {
+			sqlDB := sqlutils.MakeSQLRunner(db)
+			sqlDB.Exec(t, `CREATE TABLE foo (a INT PRIMARY KEY)`)
+			sqlDB.Exec(t, `INSERT INTO foo VALUES (1), (2), (3)`)
+			sqlDB.Exec(t, `CREATE TABLE bar (a INT PRIMARY KEY)`)
+			sqlDB.Exec(t, `INSERT INTO bar VALUES (1), (2), (3)`)
+
+			testFeed := feed(t, f, `CREATE CHANGEFEED FOR foo WITH resolved = '1s', no_initial_scan`)
+			defer closeFeed(t, testFeed)
+
+			expectResolvedTimestamp(t, testFeed)
+
+			feed, ok := testFeed.(cdctest.EnterpriseTestFeed)
+			require.True(t, ok)
+
+			sqlDB.Exec(t, `PAUSE JOB $1`, feed.JobID())
+			waitForJobStatus(sqlDB, t, feed.JobID(), `paused`)
+
+			sqlDB.Exec(t, fmt.Sprintf(`ALTER CHANGEFEED %d ADD bar WITH %s`, feed.JobID(), initialScanOption))
+
+			sqlDB.Exec(t, fmt.Sprintf(`RESUME JOB %d`, feed.JobID()))
+			waitForJobStatus(sqlDB, t, feed.JobID(), `running`)
+
+			expectPayloads := (initialScanOption == "initial_scan = 'yes'" || initialScanOption == "initial_scan")
+			if expectPayloads{
+				assertPayloads(t, testFeed, []string{
+					`bar: [1]->{"after": {"a": 1}}`,
+					`bar: [2]->{"after": {"a": 2}}`,
+					`bar: [3]->{"after": {"a": 3}}`,
+				})
+			}
+
+			sqlDB.Exec(t, `INSERT INTO bar VALUES (4)`)
+			assertPayloads(t, testFeed, []string{
+				`bar: [4]->{"after": {"a": 4}}`,
+			})
+		}
+	}
+
+	for _, initialScanOpt := range []string{"initial_scan = 'yes'", "initial_scan = 'no'", "initial_scan", "no_initial_scan"} {
+		t.Run(`kafka`, kafkaTest(testFn(initialScanOpt)))
+	}
+}
```

Comment:
nit: \n at the end.

Reply:
do you mean add newline after the function or between the last two curly braces